packages/amplify-provider-awscloudformation/lib/configuration-manager.js

```diff
@@ -30,6 +30,7 @@ async function init(context) {
 }
 
 async function configure(context) {
+  context.exeInfo = context.exeInfo || context.amplify.getProjectDetails();
```

Comment:
This change seems unrelated to the PR. What's this for?

Reply:
The provider plugin command: `amplify awscloudformation configure` is broken for a while, but I guess it's not directly used by the users, but invoked through "amplify configure project", which works,  so it's not  discovered. This fixes it the direct invocation of the configuration of the provider plugin.